src/e2e-test/features/servicenowmultisource/DesignTime.feature

```diff
@@ -48,4 +48,18 @@ Feature: ServiceNow Multi Source - Design time scenarios
     And fill Credentials section for pipeline user
     And Enter input plugin property: "startDate" with value: "start.date"
     And Enter input plugin property: "endDate" with value: "end.date"
-    Then Validate "ServiceNow Multi Source" plugin properties
\ No newline at end of file
+    Then Validate "ServiceNow Multi Source" plugin properties
+
+    @TS-SN-MULTI-DSGN-04
+    Scenario: Verify user should be able to create the valid connection using connection manager functionality for ServiceNow Multisource
+      When Open Datafusion Project to configure pipeline
+      And Select plugin: "ServiceNow Multi Source" from the plugins list as: "Source"
+      And Navigate to the properties page of plugin: "ServiceNow Multi Source"
+      And Click plugin property: "switch-useConnection"
+      And Click on the Browse Connections button
+      And Click on the Add Connection button
+      And Click on the ServiceNow Connector button
+      And Enter input plugin property: "name" with value: "ServiceNowConnection"
+      And fill Credentials section for pipeline user
```

Comment:
We need unique names for each connection create a test setup hook to create random names instead of hardcoding the name.

Reply:
done